README.rst

```diff
@@ -1,22 +1,24 @@
-Webhooks for `Open edX`_, integrating `JIRA`_ and `Github`_,
-designed to be deployed onto `Heroku`_.
+Open edX Webhooks
+=================
 
-|build-status| |coverage-status| |docs|
+Webhooks for `Open edX <http://openedx.org>`__ integrating
+`JIRA <https://openedx.atlassian.net>`__ and
+`Github <https://github.com/edx>`__. Designed to be deployed onto
+`Heroku <http://heroku.com>`__.
```

Comment:
I don't know why inline URLs are preferred to out-of-band (which I think make the text more readable in the .rst file), but OK.